src/compound.rs

```diff
@@ -286,6 +288,15 @@ impl<'a> Compound<'a> {
                 });
             }
 
+            let version = parse_version(data);
+            if parse_version(data) != 2 {
+                return Err(RtcpParseError::UnsupportedVersion(version));
+            }
+
+            if offset == 0 && parse_padding_bit(&data[offset..]) {
```

Comment:
This is not correct for an compound packet that contains a single RTCP packet. The only restriction is that only the last RTCP packet can contain padding

Reply:
The RFC explicitly says:

> The padding bit (P) should be zero for the **first** packet of a compound RTCP packet because padding should only be applied, if it is needed, to the last packet.

What you say makes more sense though and is probably what the RFC actually intends to say.

Reply:
Right, but RFC 3550 does not account for reduced size RTCP (RFC5506) where there can be only a single RTCP packet received.

Reply:
Changed all that and also added tests. Which was useful because the size check could've never triggered (and was covered before already)

Reply:
CI is also happy